bin/varnishd/cache/cache_hash.c

```diff
@@ -358,63 +358,30 @@ hsh_insert_busyobj(const struct worker *wrk, struct objhead *oh)
 /*---------------------------------------------------------------------
  */
 
-enum lookup_e
-HSH_Lookup(struct req *req, struct objcore **ocp, struct objcore **bocp)
```

Comment:
Have you spent time going back and forth because of the split or because you were reviewing the split? I  very much agree that in order to check that the split is correct this involves a lot of back and forth, and in that regard thanks for the time spent on it. I also spent a fair amount of time checking multiple times that the split was indeed equivalent.

However, when not reviewing the split itself, but rather looking at the lookup and how the result is processed separately, I find it much easier to follow their respective logic individually.

Reply:
Resolving this comment thread, the split in question is no longer part of the current patch series.